index.js

```diff
@@ -235,33 +234,35 @@ function schemaMiddleware (schema) {
 }
 
 function cors (opts) {
-  var _cors = corsify(opts)
+  opts = opts || {}
+  var headers = opts.headers || ['Content-Type', 'Accept', 'X-Requested-With']
+  var methods = opts.methods || ['PUT', 'POST', 'DELETE', 'GET', 'OPTIONS']
+  var credentials = opts.credentials || true
+  var origin = opts.origin || '*'
+
   return function (handler) {
-    var obj = {}
+    var _handler = null
     if (typeof handler === 'object') {
       var keys = Object.keys(handler)
 
       assert.equal(keys.length, 1, 'merry.cors: we can only corsify a single method per endpoint')
```

Comment:
i've been thinkin about this too, and my feeling is that it's much more approachable for folks to be able to pass in a an entire endpoint object and apply everything to an entire thing.

what i am wondering thought is how often will all endpoints need the same headers though? 'cause normally those tend to be (or at least should be) different. 

in all, lemme write a separate commit for this point and let's see how it turns out yea?